syntax = "proto3";



// Message for Order
message Order {
    int32 id = 1;
    int32 user_id = 2;
    int32 product_id = 3;
    int32 variant_id = 4;
    int32 quantity = 5;
    float total_price = 6;
    string created_at = 7; // Assuming datetime is represented as a string
}